src/pages/journal/journal.ts

```diff
@@ -32,10 +32,11 @@ export class JournalPage extends BasePageComponent implements OnInit, OnDestroy
 
   @ViewChild('slotContainer', { read: ViewContainerRef }) slotContainer;
 
-
   pageState: JournalPageState;
 
   loadingSpinner: Loading;
+  pageRefresher: Refresher;
+
   toast: Toast;
   subscription: Subscription;
   start = '2018-12-10T08:10:00+00:00';
```

Comment:
Is this still needed? Can we remove if not?

Reply:
Yes its needed so we have a reference to it and clear it when the journal has stopped loading